requiem-core/src/main/java/ladysnake/requiem/core/mixin/possession/possessed/PossessableMobEntityMixin.java

```diff
@@ -71,6 +71,13 @@ public PossessableMobEntityMixin(EntityType<? extends MobEntity> type, World wor
         super(type, world);
     }
 
+    @Override
+    protected void requiem$mobTick() {
+        this.world.getProfiler().push("mob tick");
```

Comment:
not sure what the standard is, but the capitalization looks weird to me here

Reply:
Turns out mojang just has no idea what they want. But they are using "mob tick", so we should probably do the same.